app/src/main/java/com/io/movio/ui/movielist/MovieListFragment.kt

```diff
@@ -31,7 +33,10 @@ class MovieListFragment : Fragment(), MoviesAdapter.ItemOnClickListener {
         binding.recyclerView.adapter = adapter
 
         viewModel.movieList.observe(viewLifecycleOwner) {
-            adapter.update(it)
+            when(it){
+                is Result.Success -> adapter.update(it.value)
+                is Result.Failure -> Toast.makeText(this.context , "Something went wrong!" , Toast.LENGTH_SHORT).show()
```

Comment:
it would be better if the string is in strings.xml, then you can easily reuse it on both screens